src/steps/render.js

```diff
@@ -55,29 +55,51 @@ export default async function render(state, req, res) {
   ]);
 
   // add meta
+  // (this is so complicated to keep the order backward compatible to make the diff tests happy)
+  const metadata = {
+    'og:title': meta.title,
+    'og:description': meta.description,
+    'og:url': meta.url,
+    'og:image': meta.image,
+    'og:image:secure_url': meta.image,
+    'og:image:alt': meta.imageAlt,
+    'og:updated_time': meta.modified_time,
+    'article:tag': meta.tags || [],
+    'article:section': meta.section,
+    'article:published_time': meta.published_time,
+    'article:modified_time': meta.modified_time,
+    'twitter:card': meta['twitter:card'],
+    'twitter:title': meta.title,
+    'twitter:description': meta.description,
+    'twitter:image': meta.image,
+  };
+  // remove meta with no values
+  for (const name of Object.keys(metadata)) {
+    if (!metadata[name]) {
+      delete metadata[name];
+    }
+  }
```

Comment:
note this is already the case in the current version. there an extra check in `createElement()` for that.